scraper.rb

```diff
@@ -0,0 +1,29 @@
+require 'pry'
+require 'require_all'
+require 'scraped'
+require 'scraped_page_archive/open-uri'
+
+require_rel 'lib'
+
+urls = {
+  EnglishMembersPage: [ 'http://www.althingi.is/altext/cv/en/'],
+  IcelandicMembersPage: [
+    'http://www.althingi.is/thingmenn/althingismenn/',
+    'http://www.althingi.is/thingmenn/thingmenn/varamenn-sem-sitja-a-althingi/',
+    'http://www.althingi.is/thingmenn/thingmenn/varamenn-sem-hafa-tekid-saeti/'
+  ]
+}
+
+def scraped_page_for(url)
+  Scraped::HTML.new(response: Scraped::Request.new(url: url).response)
+end
```

Comment:
this is only called from a single place at the minute, and doesn't really add much. You should probably either just inline it, or make it work with the other very similar call too (by also taking a `klass` argument)